omgeo/services/base.py

```diff
@@ -7,12 +7,6 @@
 
 import requests
 
-try:
-    # python 3
-    from urllib.parse import urlencode
```

Comment:
I see the test suite passed on Travis for both pythons. What changed to make this unnecessary for backwards support?

Reply:
These imports are unused now that https://github.com/azavea/python-omgeo/pull/53 has been merged